service/orchestrator/toe.go

```diff
@@ -81,9 +80,7 @@ func (svc *Service) ListTargetsOfEvaluation(_ context.Context, req *orchestrator
 	}
 
 	res = new(orchestrator.ListTargetsOfEvaluationResponse)
-
-	res.Toes, res.NextPageToken, err = service.PaginateStorage[*orchestrator.TargetOfEvaluation](req, svc.storage,
-		service.DefaultPaginationOpts)
+	err = svc.storage.List(&res.Toes, "", false, 0, -1, gorm.WithoutPreload())
```

Comment:
did you try `res.Toes, res.NextPageToken, err = service.PaginateStorage[*orchestrator.TargetOfEvaluation](req, svc.storage, service.DefaultPaginationOpts, gorm.WithoutPreload())`? I think this should work.